source/dub/compilers/compiler.d

```diff
@@ -183,18 +183,32 @@ interface Compiler {
 		import dub.compilers.utils : generatePlatformProbeFile, readPlatformJsonProbe;
 		import std.string : format, strip;
 
-		auto fil = generatePlatformProbeFile();
+		NativePath fil = generatePlatformProbeFile!true();
+		auto result_betterC = execute(compiler_binary ~ args ~ "-betterC" ~ fil.toNativeString());
```

Comment:
`-betterC` is called something else with GDC ,`-f-no-druntime` or something like that.

Reply:
Seems to be -fno-druntime